File: finance/cron-job.py
import json
import requests

ENDPOINT = 'https://api.cron-job.org'

headers = {
    'Authorization': 'Bearer 5Wm1mRRadgEz4kBH+kitqXcjoPqCzodIpJL0ZyGX170=',
    'Content-Type': 'application/json'
}
payload = {
    'job': {
        'enabled': False
    }
}

result = requests.patch(ENDPOINT + '/jobs/4061357', headers=headers, data=json.dumps(payload))
print(result.json())

File: finance/cron-job.php
<?   

$postdata = json_encode(array("job" => array("enabled" => true)));

$result = file_get_contents("https://api.cron-job.org/jobs/3717393", false, stream_context_create(array(
    "http" => array(
        "method" => "PATCH",
        "header" => "Content-Type: application/json" . "\r\n"
        ."Content-Length: " . strlen($postdata) . "\r\n"
        ."Authorization: Bearer RtWJSAE3zWRkwYzX70uo4lfPzfJSjLr5r7udu8Poh2c=" . "\r\n",
        "content" => $postdata,
    ),
    )));

echo $result;

?>